mule-extensions-api/src/main/java/org/mule/runtime/extension/api/annotation/param/stereotype/AllowedStereotypes.java

```diff
@@ -27,7 +28,7 @@
  * @since 1.0
  * @see StereotypeDefinition
  */
-@Target({FIELD, PARAMETER})
+@Target({FIELD, PARAMETER, TYPE})
```

Comment:
update jdoc explaining in which case it makes sense to apply this at the type level